web/site/js/vis/PixiVisualizer.js

```diff
@@ -478,19 +582,42 @@ export default class PixiVisualizer extends VObject {
     }
 
     visAddRouterTable(nodeId, extaddr) {
-        this.nodes[nodeId].addRouterTable(extaddr)
+        this.nodes[nodeId].addRouterTable(extaddr);
+        this.logNode(nodeId, `Established Router link to ${this.formatExtAddrPretty(extaddr)}`)
     }
 
     visRemoveRouterTable(nodeId, extaddr) {
-        this.nodes[nodeId].removeRouterTable(extaddr)
+        this.nodes[nodeId].removeRouterTable(extaddr);
+        this.logNode(nodeId, `Dropped Router link to ${this.formatExtAddrPretty(extaddr)}`)
     }
 
     visAddChildTable(nodeId, extaddr) {
-        this.nodes[nodeId].addChildTable(extaddr)
+        this.nodes[nodeId].addChildTable(extaddr);
+        this.logNode(nodeId, `Child ${this.formatExtAddrPretty(extaddr)} attached`)
```

Comment:
Can we use a similar format as to the router link events? E.g. `this.logNode(nodeId, 'Attached child link to ${this.formatExtAddrPretty(extaddr)}')`

Reply:
`Attached Child link to ...` is not correct because it's not the Router that attached, but the Child attached.

Reply:
What about `Added child link to...`? Or the other way around, `Router ... added` and `Router ... removed`?

Reply:
`Child Xxxx attached` reads just fine? I think it's not necessary to align with `Router` log. Thoughts?

Reply:
Changed to `Child table added: Node X(...)`